### ISSUE TITLE
训练ocr acc一直为0

### ISSUE THREAD

**Issue by xavier1001** [chineseocr/chineseocr#410]
pytorch：0.4.1  
warp_ctc : https://github.com/SeanNaren/Warp-ctc(安装的是这个)
使用作者大神提供的train-ocr代码训练，使用作者提供的train/data/ocr/0下的8张图片，loss下降，acc为0，
我现在使用10万数据进行训练，batchsize=64，lr=0.01，执行208 steps了，loss一直在降低，但acc仍然一直为0，
1407
epoch:0/10
 208/1407 [===>..........................] - ETA: 2:58:57 - loss: 9.7462 - acc: 0.0000e+00 
请问怎么解决啊？

**Comment by balabala7776** [comment]
请问你说的train-ocr在哪呢

**Comment by lonelyhwy** [comment]
同样的问题，请问能解决了吗

**Comment by pekinghk** [comment]
我也遇到了acc=0的问题，原因应该是作者在训练加载模型的时候没有加载最后一层RNN层的权重，这当然有利有弊，如果你想要充分利用作者预训练模型的训练结果，可以把模型的所有权重都加上，这样在finetune模型的时候会有比较好的初始效果。
我改动了5个地方之后可以正常训练了，可以正常训练。
1.if 'rnn.1.embedding' not in name:##不加载最后一层权重 注释掉这一行
2.将这一行:from crnn.models.crnn import CRNN 改成: from crnn.crnn_torch import CRNN
3.如果pytorch版本比较高，将crnn/util.py中loadData函数中的v.data.resize_(data.size()).copy_(data)换成v.resize_(data.size()).copy_(data)
4.如果wrap_ctc编译不通过，可以采用高版本pytorch原生的ctc,直接将ctc的引用替换成这一句就行from torch.nn import CTCLoss，或者也可以用百度的扩展ctc。
5.把训练代码中的val函数里的pred = predict(im)这一行改成pred = predict(net, im)，同样的把这个函数def predict(im)：的输入形式改成def predict(net, im): ，然后把predict函数中的这一行：preds = model(image)改成preds = net(image)。

你可以按我说的试一试。

**Comment by pekinghk** [comment]
可以看下这个 https://github.com/chineseocr/chineseocr/issues/498
